test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/SentimentPredictionTests.cs

```diff
@@ -8,6 +8,7 @@
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Runtime.FastTree;
 using Microsoft.ML.Runtime.Internal.Calibration;
+using Microsoft.ML.Runtime.Learners;
```

Comment:
The changes in this file can be reverted.  No need to change this file.

Reply:
The changes in this file are mostly refactoring that improves the style.  There are actually multiple redundant `using`s here that I am going to remove in my next commit.

Reply:
I will revert the changes - thanks!